renderer/views/torrent-list.js

```diff
@@ -19,51 +21,75 @@ function renderTorrent (state, dispatch, torrent) {
   // Background image: show some nice visuals, like a frame from the movie, if possible
   var style = {}
   if (torrent.posterURL) {
-    style['background-image'] = 'linear-gradient(to bottom, rgba(0, 0, 0, 0.5) 0%, rgba(0, 0, 0, 0) 100%), url("' + torrent.posterURL + '")'
+    style['background-image'] = `linear-gradient(to bottom, rgba(0, 0, 0, 0.5) 0%, rgba(0, 0, 0, 0) 100%), url("${torrent.posterURL}")`
   }
 
   // Foreground: name of the torrent, basic info like size, play button,
   // cast buttons if available, and delete
   var elements = [
     renderTorrentMetadata(torrent),
-    h('i.icon.delete', {
-      onclick: () => dispatch('deleteTorrent', torrent)
-    }, 'close'),
-    h('i.btn.icon.play', {
-      className: !torrent.ready ? 'disabled' : '',
-      onclick: () => dispatch('openPlayer', torrent)
-    }, 'play_arrow')
+    hx`
+      <i
+        className="icon delete"
+        onclick=${() => dispatch('deleteTorrent', torrent)}>
+        close
+      </i>
+    `,
+    hx`
+      <i
+        className="${!torrent.ready ? 'disabled btn icon play' : 'btn icon play'}"
+        onclick=${() => dispatch('openPlayer', torrent)}>
+        play_arrow
+      </i>
+    `
   ]
+
   if (state.view.chromecast) {
-    elements.push(h('i.btn.icon.chromecast', {
-      className: !torrent.ready ? 'disabled' : '',
-      onclick: () => dispatch('openChromecast', torrent)
-    }, 'cast'))
+    elements.push(hx`
+      <i
+        className="${!torrent.ready ? 'disabled btn icon chromecast' : 'btn icon chromecast'}"
+        onclick=${() => dispatch('openChromecast', torrent)}>
+        cast
+      </i>
+    `)
   }
+
   if (state.view.devices.airplay) {
-    elements.push(h('i.btn.icon.airplay', {
-      className: !torrent.ready ? 'disabled' : '',
-      onclick: () => dispatch('openAirplay', torrent)
-    }, 'airplay'))
+    elements.push(hx`
+      <i
+        className="${!torrent.ready ? 'disabled btn.icon.airplay' : 'btn.icon.airplay'}"
+        onclick=${() => dispatch('openAirplay', torrent)}>
+        airplay
+      </i>
+    `)
```

Comment:
is there no way to include some classes in the element name, like a css selector?

`i.btn.icon.airplay`